helpers/visualReady.js

```diff
@@ -0,0 +1,3 @@
+[...document.fonts].forEach(font => font.load());
+
+export const visualReady = Promise.all([document.fonts.ready]);
```

Comment:
@bearfriend let me know thoughts on this and if it's inline with what you were thinking

Reply:
Yep this looks good 👍